ProblemSolvingMentoringSession/Part 1/AbsoluteDifferenceChallenge/Solvings/AbsoluteDifferenceChallenge_Bilyk.swift

```diff
@@ -0,0 +1,19 @@
+//  Created by Maxos on 5/16/23.
+
+import Foundation
+
+final class AbsoluteDifferenceChallenge_Bilyk: AbsoluteDifferenceChallengeProtocol {
```

Comment:
This task is about abs function. Check it, and try to rewrite with this function (solution will be simpler with this function)